src/Mod/Draft/InitGui.py

```diff
@@ -87,8 +87,7 @@ def QT_TRANSLATE_NOOP(scope, text):
                         "Draft_Trimex", "Draft_Join", "Draft_Split",
                         "Draft_Upgrade", "Draft_Downgrade", "Draft_Scale",
                         "Draft_Edit", "Draft_SubelementHighlight",
-                        "Draft_WireToBSpline", "Draft_AddPoint",
-                        "Draft_DelPoint", "Draft_Shape2DView",
+                        "Draft_WireToBSpline", "Draft_Shape2DView",
```

Comment:
Just a note. This pull request as is will cause a conflict with #2970, because in this pull request the command names are moved into a new module called `draftutils.init_tools`.

Reply:
You are right, I'll correct it as soon as #2970 is merged!

Reply:
it's merged!

Reply:
Merged upstream/master to the branch!

Reply:
Carlo, remember to use the `rebase` command.
```
git checkout DraftGuiInputBugfix
git pull --rebase upstream master
```

This is so your entire branch is moved to the top of the master branch. This is better than "merging" all changes done to the master branch, because this introduces a new commit that we don't want to have in the commit history.

Reply:
Done, Thanks for the tip!